CONTRIBUTING.rst

```diff
@@ -12,21 +12,30 @@ I hope everybody knows how to work with git and github nowadays :)
 
 Workflow is pretty straightforward:
 
-  1. Clone the GitHub_ repo using the ``--recurse-submodules`` argument
+Clone the Repository:
+- 1.Open the provided GitHub link and click the "Fork" button on the upper-right of the web page to create your fork.
+- 2.Clone your forked repository using the --recurse-submodules argument to ensure submodules are also cloned.
 
-  2. Setup your machine with the required development environment
+Set Up Development Environment:
+- 1.Configure your machine with the necessary development environment as specified in the project's documentation.
 
-  3. Make a change
+Make Changes:
+- 1.Implement the desired changes in your local clone.
 
-  4. Make sure all tests passed
+Run Tests:
+- 1.Ensure all tests pass before proceeding to the next step.
 
-  5. Add a file into the ``CHANGES`` folder, named after the ticket or PR number
+Update Changes File:
+- 1.Add a file to the CHANGES folder, naming it after the ticket or pull request number.
 
-  6. Commit changes to your own aiohttp clone
+Commit Changes:
+- 1.Commit your changes to your own aiohttp clone.
 
-  7. Make a pull request from the github page of your clone against the master branch
+Create Pull Request:
+- 1.Make a pull request from the GitHub page of your clone against the master branch of the original repository.
+Optional Backport:
 
-  8. Optionally make backport Pull Request(s) for landing a bug fix into released aiohttp versions.
```

Comment:
This line should probably be changed though to better reflect current practices.

Maybe something like:
Once merged, backports may be created automatically by a bot. If the bot fails to do this, please follow the instructions from the bot to complete the backport PRs.

Reply:
Well, i will check that, thanks for the information!

Reply:
That only works if somebody reviewing the PR puts the labels on. The contributors might still want to optionally offer manually-crafted backport PRs. (or semi-manually — we have a config for the CPython's `cherry-picker` tool!)

Reply:
Yeah, I'm just thinking it's usually safer to wait for the bot. A maintainer should add the label when they believe a backport is needed. Surely cherry picking won't work well if done before merging anyway, as they'd be picking commits from the branch, which won't end up on master (there will just be a squash commit instead).

Reply:
Note that the bot experiences instability because of https://github.com/aio-libs/aiohttp/issues/4581, that I haven't really had time to tackle properly. I think it needs to be prioritized but there's a lot on my plate right now. A workaround is to re-apply the label (but for that, one must notice the failure first).